libs/shared/providers/src/wagmi/components/OpenAccountModalButton.tsx

```diff
@@ -38,8 +37,9 @@ export const OpenAccountModalButton = ({
           return (
             <Button
               {...(disconnectedProps
-                ? (mergeDeepRight(props, disconnectedProps) as any)
+                ? (mergeDeepRight(props, disconnectedProps) as ButtonProps)
                 : props)}
+              variant={'nav'}
```

Comment:
placing the variant here won't allow any override, it needs to be above `...(disconnectedProps ? ...)`